trigger_aws.yml

```diff
@@ -9,33 +8,35 @@ phases:
       # Device Farm support two major versions of Python, each with one minor version: 2 (minor version 2.7), and 3 (minor version 3.7.4).
       # The default Python version is 2, but you can switch to 3 by setting the following variable to 3:
       - export PYTHON_VERSION=3
-      
-      # By default, Appium server version used is 1.7.2.
-      # You can switch to an alternate supported version from 1.6.5, 1.7.1, 1.7.2, 1.8.0 or 1.8.1 by using a command like "avm 1.7.1"
-      # OR
-      # To install a newer version of Appium use the following commands:
+      # This command will install your dependencies and verify that they're using the proper versions that you specified in your requirements.txt file. Because Device Farm preconfigures your environment within a
+      # Python Virtual Environment in its setup phase, you can use pip to install any Python packages just as you would locally.
+      - cd $DEVICEFARM_TEST_PACKAGE_PATH
+      - . bin/activate
+      - pip install -r requirements.txt
```

Comment:
- I think only `pip install -r requirements.txt` install the python required dependencies.

- what is the purpose of `. bin/activate` command, need a separate doc-string.
- Overall doc-string can be improved.
- is there any commend mention to verify? that check the required dependencies are installed or not.

Reply:
bin/activate command will activate this virtual environment
and i have updated the doc-string